pkg/cli/mirror/mirror.go

```diff
@@ -223,6 +223,22 @@ func (o *MirrorOptions) Run(cmd *cobra.Command, f kcmdutil.Factory) (err error)
 			return err
 		}
 
+		prevAssociations, err := image.ConvertToAssociationSet(meta.PastAssociations)
```

Comment:
This block of code and the one below are identical. It would be nice if we can reduce the duplication here. Not a big deal. Just a nit.

Reply:
I will try to reduce the duplication here. I think there is some duplication between mirror to disk and mirror to mirror workflow options.